src/main/kotlin/venus/glue/Driver.kt

```diff
@@ -155,4 +155,8 @@ import kotlin.browser.window
         }
         Renderer.updateRegister(id, sim.getReg(id))
     }
+
+    @JsName("setDisplay") fun render_setDisplay(dis_type: String) {
```

Comment:
Change this to camelCase? Also maybe a better name would be `setRegisterDisplay` or something?